ibm_catalog.json

```diff
@@ -117,6 +241,14 @@
               },
               "required": true
             },
+            {
+              "key": "existing_monitoring_crn",
+              "required": true
+            },
+            {
+              "key": "existing_scc_wp_crn",
+              "required": true
+            },
```

Comment:
I would not force require both of these - the user only has to pass one or the other. I would keep `existing_monitoring_crn` as required but ensure description mentions they can set it to `null` and use `existing_scc_wp_crn` instead if they don't have a monitoring instance